src/common/kafka.spec.ts

```
@@ -17,3 +17,147 @@ describe('stage config', () => {
+
+    const { produceMessage, disconnectProducer } = await import('./kafka');
+    const producePromise = produceMessage('topic-a', { foo: 1 });
+    const disconnectPromise = disconnectProducer();
+
+    await jest.advanceTimersByTimeAsync(5_000);
+
+    await expect(disconnectPromise).resolves.toBeUndefined();
+    expect(mockDisconnect).toHaveBeenCalledTimes(1);
+
+    resolveConnect();
+    await expect(producePromise).rejects.toThrow('Kafka producer is shutting down');
```

ESLint

[error] Replace `'Kafka·producer·is·shutting·down'` with `⏎······'Kafka·producer·is·shutting·down',⏎····`

(prettier/prettier)